components/line/actions/send-a-push-message/send-a-push-message.mjs

```diff
@@ -0,0 +1,48 @@
+import line from "../../line.app.mjs";
+
+export default {
+  name: "Send a Push Message",
+  description: "Sends a push message to a user, group, or room at any time.",
+  key: "line-send-a-push-message",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    line,
+    channelAccessToken: {
+      propDefinition: [
+        line,
+        "channelAccessToken",
+      ],
+    },
+    to: {
+      label: "To",
+      type: "string",
+      description: "The id of user, group, or room the message will be sent to.",
+    },
+    message: {
+      propDefinition: [
+        line,
+        "message",
+      ],
+    },
+    notificationDisabled: {
+      propDefinition: [
+        line,
+        "notificationDisabled",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const client = this.line.createLineClient(this.channelAccessToken);
+
+    const response = client.pushMessage(this.to, {
+      type: "text",
+      text: this.message,
```

Comment:
**Question**

According to the [docs](https://developers.line.biz/en/reference/messaging-api/#send-push-message), this parameter is an `array` of `strings`.
Does the API accept a `string` parameter?

If not, it is OK to send only one message - also simplifies user experience.
The request should be changed to:

```suggestion
      text: [
        this.message,
      ],
```

Reply:
The API accept a string, dont need change to a array.